sdk/iothub/Microsoft.Azure.Management.IotHub/src/Microsoft.Azure.Management.IotHub.csproj

```diff
@@ -15,7 +15,7 @@
     <AssemblyName>Microsoft.Azure.Management.IotHub</AssemblyName>
     <Version>4.2.0</Version>
     <PackageTags>Microsoft Azure IotHub;IotHub management;IotHub;</PackageTags>
-    <PackageReleaseNotes>Regenerated from an updated 2021-07-02 swagger.</PackageReleaseNotes>
+    <PackageReleaseNotes>Regenerated from an updated 2022-04-30-preview swagger.</PackageReleaseNotes>
```

Comment:
Please also update the version here from `4.2.0` to `4.3.0-preview`.
Also update the version in  AssemblyInfo.cs file

Reply:
can't postfix with .preview in AssemblyInfo.cs, I made the change to 4.3.0.0